runtime/src/lib.rs

```diff
@@ -78,11 +78,11 @@ pub use sp_runtime::{Perbill, Permill};
 pub type BlockNumber = u32;
 
 /// Alias to 512-bit hash when used in the context of a transaction signature on the chain.
-pub type Signature = MultiSignature;
+pub type Signature = account::MultiSignature;
 
 /// Some way of identifying an account on the chain. We intentionally make it equivalent
 /// to the public key of our transaction signing scheme.
-pub type AccountId = <<Signature as Verify>::Signer as IdentifyAccount>::AccountId;
+pub type AccountId = account::AccountId20;
```

Comment:
Why change this one? Letting the type system calculate our AccountId type for us, means we won't have to update this line if things change in the future.

Reply:
Yes, that was mostly legacy because I implemented the AccountId before the Signer. I'm changing it, thanks.